src/containers/Register/index.js

```diff
@@ -45,107 +47,111 @@ class Register extends Component {
-          underlineStyle={{ height: 0 }}
-          errorStyle={{ textAlign: 'left', color: red600 }}
-          onChange={this.onChange} />
-        <TextField
-          type='text'
-          id='address-1'
-          name='address-1'
-          floatingLabelText='Address 1'
-          fullWidth={true}
-          floatingLabelFocusStyle={{ color: lightBlue400 }}
-          underlineFocusStyle={{ borderColor: lightBlue400 }}
-          underlineStyle={{ height: 0 }}
-          errorStyle={{ textAlign: 'left', color: red600 }}
-          onChange={this.onChange} />
-        <TextField
-          type='text'
-          id='address-2'
-          name='address-2'
-          floatingLabelText='Address 2'
-          fullWidth={true}
-          floatingLabelFocusStyle={{ color: lightBlue400 }}
-          underlineFocusStyle={{ borderColor: lightBlue400 }}
-          underlineStyle={{ height: 0 }}
-          errorStyle={{ textAlign: 'left', color: red600 }}
-          onChange={this.onChange} />
-        <TextField
-          type='text'
-          id='city'
-          name='city'
-          floatingLabelText='City'
-          fullWidth={true}
-          floatingLabelFocusStyle={{ color: lightBlue400 }}
-          underlineFocusStyle={{ borderColor: lightBlue400 }}
-          underlineStyle={{ height: 0 }}
-          errorStyle={{ textAlign: 'left', color: red600 }}
-          onChange={this.onChange} />
-        <SelectField
-          id='state'
-          name='state'
-          floatingLabelText='State'
-          fullWidth={true}
-          floatingLabelFocusStyle={{ color: lightBlue400 }}
-          underlineFocusStyle={{ borderColor: lightBlue400 }}
-          underlineStyle={{ height: 0 }}
-          value={state}
-          onChange={(e, key, val) => this.onSelectChange(e, key, val)}>
-          {this.renderStates()}
-        </SelectField>
-        <TextField
-          type='text'
-          id='zipcode'
-          name='zipcode'
-          floatingLabelText='Zip Code'
-          fullWidth={true}
-          floatingLabelFocusStyle={{ color: lightBlue400 }}
-          underlineFocusStyle={{ borderColor: lightBlue400 }}
-          underlineStyle={{ height: 0 }}
-          errorStyle={{ textAlign: 'left', color: red600 }}
-          onChange={this.onChange} />
-        <FlatButton
-          type='submit'
-          label='Submit'
-          backgroundColor={lightBlue400}
-          hoverColor={lightBlue600}
-          labelStyle={{ color: white, fontWeight: 300 }}
-          fullWidth={true}
-          style={{ margin: '12px 0' }} />
-      </form>
+      <Row>
+        <Column small={12} medium={8} large={6} offset={{ medium: 2, large: 3}}>
+          <form onSubmit={this.handleFormSubmit}>
+            <TextField
+              type='text'
+              id='name'
+              name='name'
+              floatingLabelText='Restaurant Name'
+              fullWidth={true}
+              floatingLabelFocusStyle={{ color: lightBlue400 }}
```

Comment:
Can we add a classname to these `material-u` fields and then move the style settings to a `.scss` file?

Reply:
No, we cannot. we can add className prop to the material-ui component, but how material-ui set the style is using inline style. So, we cannot set css property with class if the property is set by inline style from material-ui component.